lib/libmp/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
This is already added in Zephyr LICENSES directory. I think the SPDX identifiers in the file is enough